server/views/pages/movePrisoner/movePrisonerCompleted.njk

```diff
@@ -28,3 +28,8 @@
     </div>
   </div>
 {% endblock %}
+
+{% block pageScripts %}
+  <script src="/assets/js/googleAnalyticsEvents.js"></script>
```

Comment:
I take it that the only way we can see these pages is if the move has been completed?